src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_common.py

```diff
@@ -83,12 +85,15 @@ def firewall_rule_create_func(client, resource_group_name, server_name, firewall
 def migration_create_func(cmd, client, resource_group_name, server_name, properties, migration_name=None):
 
     subscription_id = get_subscription_id(cmd.cli_ctx)
-
+    properties_filepath = os.path.join(os.path.abspath(os.getcwd()), properties)
```

Comment:
What if `properties` is absolute path?

Actually the proper way to define a file_path param in CLI is use `type=file_type, completer=FilesCompleter()`
See example in storage module:
https://github.com/Azure/azure-cli/blob/228fd1c8e749a3fbbf32593e47bdc8ca91ddb9b2/src/azure-cli/azure/cli/command_modules/storage/_params.py#L486-L489

Reply:
os.path.join() will join only if properties is a filename and not otherwise. I tested this with properties file being a filename and absolute path and both worked .